requirements.txt

```diff
@@ -5,11 +5,13 @@ onnx
 onnxoptimizer
 onnxruntime
 opencv-python
+pasteboard ; sys_platform=="darwin"
 Pillow
 protobuf==3.16.0
 pylint
 pyright==1.1.249
 pytest
+pywin32 ; sys_platform=="win32"
```

Comment:
You'll also need to add the extra dependencies to dependencies.ts in order for them to be auto-installed on startup. Requirements.txt is just for local development installation.

Reply:
It's [this](https://github.com/joeyballentine/chaiNNer/blob/main/src/common/dependencies.ts) file by the way.